api/lib/notificationController.js

```diff
@@ -0,0 +1,81 @@
+const database = require("./database");
```

Comment:
I think we should make a folder called `notifications`. Otherwise the `lib` folder gets too messy